tools/update_goversion/main.go

```diff
@@ -147,8 +145,11 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	for file, hash := range hashes {
+		fmt.Printf("%s  %s\n", hash, file)
+	}
 
-	zeroPatchVersion := removeZeroPatch(version)
+	_ = removeZeroPatch(version)
```

Comment:
I wondered this to. I didn't want to get rid of the function in case upstream changes their mind on the zero versioning, but that is what a git history is for :laughing: so I have removed it